packages/kv-asset-handler/package.json

```diff
@@ -50,7 +50,7 @@
 		"service-worker-mock": "^2.0.5"
 	},
 	"engines": {
-		"node": ">=16.13"
+		"node": ">=18.0.0"
```

Comment:
We should bump the version of this package to 0.4.0 as this is arguably a breaking change.

Reply:
b9b42f725

Reply:
b9b42f725